### ISSUE TITLE
2.1: Support specifying VPC ID when creating kubes

### ISSUE THREAD

**Issue by bfosberry** [supergiant/control#341]
When creating a kube through ui/kubes, add an option to specify a VPC ID in which to provision the cluster

**Comment by gopherstein** [comment]
We do provide the capability currently with 2 known risks that need to be further flushed out. 

https://github.com/supergiant/supergiant/blob/master/pkg/model/kube.go#L80

1. We have been improving this for a while and a user "carlos" in slack is even deploying to production. But we still feel like deploying into a vpc needs more vetting. We use a flag to tell SG that the VPC does not belong to it.. 

2. You must specify as many of the other options as you can when deploying into a vpc. like ip range etc.. 

Basically, it exists but needs vetting, and could probably use a logic pass....

**Comment by gopherstein** [comment]
User should be able to select a VPC from a list in the UI or enter a VPC number in the api.

**Comment by gopherstein** [comment]
This feature is active in 2.0
